model/simulationtests/electric_equipment_ITE.rb

```diff
@@ -0,0 +1,115 @@
+require 'openstudio'
+require 'lib/baseline_model'
+
+model = BaselineModel.new
+
+#make a 2 story, 100m X 50m, 10 zone core/perimeter building
+model.add_geometry({"length" => 100,
+                    "width" => 50,
+                    "num_floors" => 1,
+                    "floor_to_floor_height" => 4,
+                    "plenum_height" => 1,
+                    "perimeter_zone_depth" => 10})
```

Comment:
@KaiyuSun Seems like you're really only using 2 spaces to add ITE after. Unless there's a reason to have more spaces, perhaps you should consider making this a 2 story, 2 zone building by setting `perimeter_zone_depth` to 0.
This matters because it'll run faster in E+ if you have a smaller building.

Reply:
@jmarrec Thank you for the edits and comments! I've updated the number of spaces and committed the changes. I've also tested your commits on my branch and it works well.